### ISSUE TITLE
Update `binding-positions` rule for Lit v2

### ISSUE THREAD

**Issue by stramel** [43081j/eslint-plugin-lit#98]
In Lit v2 you can achieve some of these invalid binding positions by using `import { html } from 'lit/static-html.js';
`.

We should probably warn about the binding positions if they're not utilizing the static html function

**Comment by aboutsimon** [comment]
Same issue with [no-invalid-html](https://github.com/43081j/eslint-plugin-lit/blob/master/docs/rules/no-invalid-html.md), when trying to use e.g. the code from the [lit docs](https://lit.dev/docs/templates/expressions/#static-expressions)

```js
  render() {
    return html`
      <${this.tag} ${this.activeAttribute}?=${this.active}>
        <p>${this.caption}</p>
      </${this.tag}>`;
  }
```

**Comment by 43081j** [comment]
that one in particular is because our placeholders are currently comments except in the case of attributes, then they are placeholder attributes.

we need to implement better placeholder/substitution to account for tags/attributes/etc.

i was hoping peter on the lit team would finish the analyzer he started building first, so we could just use the same analysis here.
